about.html

```diff
@@ -0,0 +1,304 @@
+                            </p>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+        <div>
+            <!-- ---------------------Middle section about page  -->
+            <div class="container-fluid about_middle_section">
+                <div class="row">
+                    <div class="col-4"></div>
+                    <div class="col-8">
+                        <div class="about_middle_container">
+                            <h2>RR Company Logo</h2>
+                            <div class="row">
+                                <div class="col-2"></div>
+                                <div class="col-10">
+                                    <div class="about_divider"></div>
+                                </div>
+                            </div>
+                        </div>
+
+
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-2"></div>
+                    <div class="col-8 about_paragraph_container">
+                        <p>
+                          Organising the event is very challenging when you are not an event planner.
+                           But the biggest 2 challenges are getting everyone to RSVP on time and getting 
+                           the right guest speakers. 
+                            <span class="hide_mobile">
+                                We started with venue costs, asking many venues for a bunch of stuff 
+                                that we'd need on the day. And found the best price plus ambience.
+                                 Then we moved on to march quotes. Deciding what merchandise
+                                 we want and getting costs. Then ran a competition for artwork for 
+                                 the shirt and hoodies we planned to make. 
+                                 We then asked a bunch of external people if they would be interested in
+                                  speaking and shortlisted the right people and topics who responded as being 
+                                  
+                                  available.  We then asked internally for speakers. 
+                                 After we had a list of speakers, we then met regularly to check on the status of the out standing items, and to fi out agenda for the day. 
+                                  And the week before was finalising, confirming and executing the event. Afterwards we also sent out a feedback form to our company to get positive and negative feedback to improve the next event.
+                        </span></p>
+
+                    </div>
+                </div>
+                <div class="row border_line">
+                    <div class="col-4"></div>
+                    <div class="col-5 about_midlle_image">
+                        <img src="./images//rrtb.avif" alt="logo bottom">
```

Comment:
- [ ] This logo image is way bigger than the original design.
- [ ] Also the original design logo image has the exact top and bottom margin defined but in your design, this logo image doesn't have a bottom margin at all. so kindly consider fixing it.
![h](https://user-images.githubusercontent.com/96683843/226443896-da7be477-d23a-44d4-9bc3-20a0315da5c3.jpg)
Original design
![g](https://user-images.githubusercontent.com/96683843/226443954-8390fa0c-08a9-44ab-8cca-81e069f964eb.jpg)